SonarQube.TeamBuild.PreProcessor/SonarWebService.cs

```diff
@@ -323,10 +319,22 @@ private void DownloadServerVersion()
                 GetPropertyValue(settings, t);
             }
 
-            // http://jira.sonarsource.com/browse/SONAR-5891 or when C# plugin is not installed
-            if (!settings.ContainsKey("sonar.cs.msbuild.testProjectPattern"))
+
+            return checkTestProjectPattern(settings);
+        }
+
+        private Dictionary<string, string> checkTestProjectPattern(Dictionary<string, string> settings)
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png 'Severity: MINOR') Rename method 'checkTestProjectPattern' to match camel case naming rules, consider using 'CheckTestProjectPattern'. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://next.sonarqube.com/sonarqube/coding_rules#rule_key=csharpsquid%3AS100)